topdown/http.go

```diff
@@ -149,6 +149,54 @@ func canonicalizeHeaders(headers map[string]interface{}) map[string]interface{}
 	return canonicalized
 }
 
+//getCertPool returns a cert pool with the given cert file, env var
+//and speicfication for using the system certs (which defaults to true if none)
+//other supplied. Returns the configured cert pool or nil if adding to the pool errors
```

Comment:
Nit: ` (which defaults to true if none)` > ` (which defaults to true if none other supplied)`